components/brave_new_tab_ui/containers/newTab/settings/braveToday/sources.tsx

```diff
@@ -92,6 +102,12 @@ export default function Sources (props: SourcesProps) {
       return result
     }
     for (const publisher of Object.values(props.publishers)) {
+      // If the publisher has a locale (which can only happen in the V2 API) and
+      // it doesn't include the current locale, skip over it.
+      if (publisher.locales.length !== 0 && !publisher.locales.includes(locale)) {
+        continue
+      }
```

Comment:
This seems fine for now but as the list grows we might want to filter on the browser side and send them on-demand for the client-side to avoid sending a huge list. That would add a fair amount of complexity, so completely fine to do it this way until we see a problem.

Reply:
Yeah let's leave it until we see it becoming a problem :smile: